crlite_query/__init__.py

```diff
@@ -399,11 +399,11 @@ def __init__(self, *, name, issuer, cert_id, crlite_db, not_before, not_after):
             self.state = "Not Enrolled"
             return
 
-        if crlite_db.latest_covered_date() < not_before:
+        if crlite_db.filter_date() < not_before:
             self.state = "Too New"
             return
 
-        if crlite_db.latest_covered_date() > not_after:
+        if crlite_db.filter_date() > not_after:
```

Comment:
I think this could go either way, for date of effect. It should maybe just be clearly documented.